addons/popochiu/engine/objects/graphic_interface/templates/9_verb/9_verb_commands.gd

```diff
@@ -94,35 +98,55 @@ func look_at() -> void:
 	if E.clicked:
 		await C.player.face_clicked()
 	
-	if I.clicked:
-		I.clicked.set_active(true)
-		return
-	
 	await C.player.say("I have nothing to say about that")
 
 
 ## Called when [code]E.current_command == Commands.PULL[/code] and [code]E.command_fallback()[/code]
 ## is triggered.
 func pull() -> void:
-	C.player.say("I don't want to pull that")
+	await C.player.say("I don't want to pull that")
 
 
 ## Called when [code]E.current_command == Commands.GIVE[/code] and [code]E.command_fallback()[/code]
 ## is triggered.
 func give() -> void:
-	C.player.say("What?")
+	await C.player.say("What?")
 
 
 ## Called when [code]E.current_command == Commands.TALK_TO[/code] and
 ## [code]E.command_fallback()[/code] is triggered.
 func talk_to() -> void:
-	C.player.say("Emmmm...")
+	await C.player.say("Emmmm...")
 
 
 ## Called when [code]E.current_command == Commands.USE[/code] and [code]E.command_fallback()[/code]
 ## is triggered.
 func use() -> void:
-	C.player.say("What?")
+	if I.active and E.clicked:
+		use_item_on(I.active, E.clicked)
+	elif I.active and I.clicked and I.active != I.clicked:
+		use_item_on(I.active, I.clicked)
+	elif I.clicked:
+		match I.clicked.last_click_button:
+			MOUSE_BUTTON_LEFT:
+				I.clicked.set_active(true)
+			MOUSE_BUTTON_RIGHT:
+				# TODO: I'm not sure this is the right way to do this. Maybe GUIs should capture
```

Comment:
Sure.

My doubt has to do with the fact that within a function that is supposed to be a fallback, the execution of a command (`I.clicked.handle_command(MOUSE_BUTTON_LEFT)`) is triggered again.

Reply:
Aaaah, I see... yes, your concern makes sense, but since it's not at engine level, but at the game level (the interface is copied over as part of the game) I think the logic is context-dependent enough to make an endless loop avoidable.

So far I would leave it like it is now.